projects/commudle-admin/src/app/app.module.ts

```diff
@@ -1,195 +1,197 @@
-
-
-export function initApp(appInitService: AppInitService) {
-  return () => appInitService.initializeApp();
-}
-
-@NgModule({
-  declarations: [
-    AppComponent,
-    OrganizerCommunitiesListComponent,
-    CommunityComponent,
-    HomeComponent,
-    FillDataFormComponent,
-    LogoutComponent,
-    SidebarMenuComponent,
-    BasicUserProfileComponent,
-    SpeakerResourceFormComponent,
-    EventCardComponent,
-    EditProfileComponent,
-    CommunityBuildCardComponent,
-    AboutComponent,
-    FeaturesComponent,
-    CommunitiesComponent,
-    SwUpdateComponent,
-    HomeCommunityCardComponent,
-    HomeEventsComponent,
-    HomeEventsCardComponent,
-    HomeLabsComponent,
-    HomeCommunitiesComponent,
-    HomeBuildsComponent,
-    HomeBuildsCardComponent
-  ],
-  imports: [
-    AppRoutingModule,
-    BrowserModule.withServerTransition({appId: 'serverApp'}),
-    BrowserAnimationsModule,
-    HttpClientModule,
-    FontAwesomeModule,
-    FormsModule,
-    ReactiveFormsModule,
-    EditorModule,
-    AppSharedComponentsModule,
-    SharedComponentsModule,
-    ReusableComponentsModule,
-    SharedPipesModule,
-    UserChatsModule,
-    CommunityChannelsModule,
-    SkeletonScreensModule,
-
-
-    // external service modules
-    LibErrorHandlerModule,
-    NgxMaterialTimepickerModule,
-
-    // Nebula modules
-    NbThemeModule.forRoot({name: 'default'}),
-    NbLayoutModule,
-    NbEvaIconsModule,
-    NbSidebarModule,
-    NbButtonModule,
-    NbCardModule,
-    NbRouteTabsetModule,
-    NbInputModule,
-    NbCheckboxModule,
-    NbSelectModule,
-    NbDatepickerModule.forRoot(),
-    NbRadioModule,
-    NbUserModule,
-    NbContextMenuModule,
-    NbMenuModule.forRoot(),
-    NbIconModule,
-    NbListModule,
-    NbTooltipModule,
-    NbWindowModule.forRoot(),
-    NbAccordionModule,
-    NbBadgeModule,
-    NbTabsetModule,
-    NbToastrModule.forRoot(),
-    NbFormFieldModule,
-    NbDialogModule.forRoot(),
-    NbSpinnerModule,
-    NbTimepickerModule.forRoot(),
-
-    // Other external npm modules
-    Ng2CompleterModule,
-    Ng2SmartTableModule,
-    ServiceWorkerModule.register('ngsw-worker.js', {enabled: environment.production}),
-  ],
-  providers: [
-    AppInitService,
-    Title,
-    CookieService,
-    NbSidebarService,
-    PrismJsHighlightCodeService,
-    {
-      provide: APP_INITIALIZER,
-      useFactory: initApp, deps: [AppInitService],
-      multi: true
-    },
-    {
-      // TODO move the interceptors to a common barrel file if needed
-      // https://angular.io/guide/http#provide-the-interceptor
-      provide: HTTP_INTERCEPTORS,
-      useClass: AuthTokenInterceptor,
-      multi: true
-    },
-    {
-      provide: HTTP_INTERCEPTORS,
-      useClass: ApiParserResponseInterceptor,
-      multi: true
-    }
-  ],
-  bootstrap: [AppComponent]
-})
-export class AppModule {
-}
+import {AppRoutingModule} from './app-routing.module';

```

Comment:
Why is this file recreated? Just curious

Reply:
I think that was because of some line ending issue on my side. I've fixed it now, so wouldn't happen again!